webapp/src/js/controllers/reports-content.js

```diff
@@ -74,6 +75,11 @@ var _ = require('underscore');
         }
       };
 
+      $scope.labelIsIDorName = (label) => {
+        let val = $translate.instant(label);
+        return val === 'ID' || val === 'Name';
```

Comment:
This won't work in any language other than English. I think you'd be better of checking the `label` itself. Perhaps something like...

```js
      $scope.labelIsIDorName = (label) => {
        return label.endsWith('.patient_id') || label.endsWith('.patient_uuid') || label.endsWith('.patient_name');
      };
```

I think this will work more reliably.